command/resource/client/grpc-config.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package client
+
+import (
+	"os"
+)
+
+const (
+	// GRPCAddrEnvName defines an environment variable name which sets the gRPC
+	// address for consul connect envoy. Note this isn't actually used by the api
+	// client in this package but is defined here for consistency with all the
+	// other ENV names we use.
+	GRPCAddrEnvName = "CONSUL_GRPC_ADDR"
```

Comment:
Are you intending to add other generic env vars such as `CONSUL_NAMESPACE` in follow-up PRs?

Reply:
Yes, that's the plan, the file of `grpc-config.go` hosts all of the config variables.